build-cs/composer.json

```
@@ -0,0 +1,16 @@
+{
+  "require-dev": {
+		"php-stubs/wordpress-tests-stubs": "^6.3.0",
+		"phpstan/extension-installer": "^1.3.1",
+    "phpstan/phpstan": "^1.10",
+    "phpstan/phpstan-deprecation-rules": "^1.1",
+    "phpstan/phpstan-phpunit": "^1.3",
+		"swissspidy/phpstan-no-private": "^v0.2.0",
+    "szepeviktor/phpstan-wordpress": "^1.3.0"
+  },
+  "config": {
+    "allow-plugins": {
+      "phpstan/extension-installer": true
+    }
+  }
+}
```

TAB-s are mixed with spaces
```suggestion
{
  "require-dev": {
    "php-stubs/wordpress-tests-stubs": "^6.3.0",
    "phpstan/extension-installer": "^1.3.1",
    "phpstan/phpstan": "^1.10",
    "phpstan/phpstan-deprecation-rules": "^1.1",
    "phpstan/phpstan-phpunit": "^1.3",
    "swissspidy/phpstan-no-private": "^v0.2.0",
    "szepeviktor/phpstan-wordpress": "^1.3.0"
  },
  "config": {
    "allow-plugins": {
      "phpstan/extension-installer": true
    }
  }
}
```